README.md

```diff
@@ -23,6 +23,22 @@ sdk = TravelTimeSdk('YOUR_APP_ID', 'YOUR_APP_KEY')
 ### Concurrent calls 
 Each method below has its own asynchronous version.
 
+#### Example:
+```python
+from datetime import datetime
+
+from traveltimepy import Driving, Coordinates, TravelTimeSdk
+
+sdk = TravelTimeSdk('YOUR_APP_ID', 'YOUR_APP_KEY')
+
+res = await sdk.time_map_async(
+    coordinates=[Coordinates(lat=51.507609, lng=-0.128315), Coordinates(lat=51.517609, lng=-0.138315)],
+    arrival_time=datetime.now(),
+    transportation=Driving()
+)
+```
+
+
```

Comment:
That's not a proper example, You need to show how to make 100x100 matrix by utilizing concurrency. Or something useful. Just dropping the async method ain't enough. All examples should be easy to understand and useful, not just code snippets which doesn't do anything.

Reply:
Look at benchmarks folder. There I have these examples.